src/app/src/sensorUtils.js

```diff
@@ -109,9 +126,9 @@ export const getSensorByProp = (prop, value) =>
 export function calculateOverallSensorRating(sensorRatings) {
     const totalValue = values(sensorRatings).reduce((acc, r) => acc + r, 0);
 
-    if (totalValue >= 4) {
+    if (totalValue >= 2.5) {
```

Comment:
Good call adjusting this until IBI is a thing to incorporate, or not. These current thresholds felt a little generous in the app (e.g. bad + fair + good + good => good). Wondering why x.5 thresholds were picked?

Reply:
Yeah, in retrospect they do seem generous. Happy to modify them. How about this?

x >= 3: Good
3 > x >= 1.5: Fair
x < 1.5: Bad

Reply:
I like it! Cutting our rivers some slack at the bottom end of the curve...

Reply:
Updated